Sprint-2/3-mandatory-implement/3-to-pounds.js

```diff
@@ -4,3 +4,25 @@
 // You will need to declare a function called toPounds with an appropriately named parameter.
 
 // You should call this function a number of times to check it works for different inputs
+
+let toPounds = function (str) {
+  const penceStringWithoutTrailingP = str.substring(
+    0,
+    penceString.length - 1 // take only 399
```

Comment:
penceString variable is a variable defined outside, Maybe you meant str? Check this please